.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,7 +33,7 @@ implementation proposal in the referenced issues.
 <!-- Make sure that this pull request is complete. -->
 
 - [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
-- [ ] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
+- [ ] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`. If yes, the target branch is set to `master`, otherwise the target branch is set to `develop`.
```

Comment:
This implies that non-compatible changes may go to `develop`, which is not the case. Patches go to `master`. New features, API additions and DB changes go to `develop`. Breaking changes are still not allowed.

Reply:
Please check the updated file.